package.json

```diff
@@ -20,8 +20,9 @@
     "test-check-coverage": "npm run test-coverage && nyc check-coverage",
     "test-cloud": "mochify --wd --no-detect-globals --timeout=10000",
     "test-coverage": "nyc --all --reporter text --reporter html --reporter lcovonly npm run test-node",
-    "test": "npm run lint && npm run test-node && npm run test-headless",
-    "bundle": "browserify --no-detect-globals -s FakeTimers -o fake-timers.js src/fake-timers-src.js",
+    "test-esm": "node -r esm test/fake-timers-esm-test.js",
```

Comment:
Where does this file come from? It is not added in any of these commits and it is not present in the `master` branch. Not sure how this test can pass ... 🤔

Reply:
yeah sorry looks like i forgot to push it, will add it today